Assets/MixedRealityToolkit.Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubRegisteredServiceProvidersProfile.asset

```diff
@@ -16,9 +16,9 @@ MonoBehaviour:
   configurations:
   - componentType:
       reference: Microsoft.MixedReality.Toolkit.Extensions.SceneTransitions.SceneTransitionService,
-        Assembly-CSharp
+        Microsoft.MixedReality.Toolkit.Extensions.SceneTransitionService
     componentName: Scene Transition Service
     priority: 10
     runtimePlatform: 15
-    configurationProfile: {fileID: 11400000, guid: 23872a85e06a8a54f9381e062152f891,
+    configurationProfile: {fileID: 11400000, guid: 45c20fade71a3ec418860b757ff5a5ce,
```

Comment:
I think the Examples Hub has it's own specific profile, instead of using the default here

```suggestion
    configurationProfile: {fileID: 11400000, guid: 23872a85e06a8a54f9381e062152f891,
```

Reply:
hm i have a custom profile but my files don't show as changed.

Reply:
Ah! I was wrong. Fixing